docs/content/how-to/powervs/create-cluster-powervs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Create PowerVS Hypershift Cluster
+---
+
+# Create PowerVS Hypershift Cluster
+
+Creating Hypershift cluster in IBM Cloud PowerVS service.
+
+## Prerequisites
+
+Please see [prerequisites](./prerequisites-and-env-guide.md/#prerequisites) before setting up the cluster
+
+## Creating the Cluster
+
+Use the `hypershift create cluster powervs` command:
+
+    ./bin/hypershift create infra powervs --base-domain BASEDOMAIN
+        --resource-group RESOURCE_GROUP \
+        --pull-secret PULL_SECRET \
+        --region POWERVS_REGION \
+        --zone POWERVS_ZONE \
+        --vpc-region VPC_REGION \
+        --node-pool-replicas=2
+
+E.g.:
+
+      ./bin/hypershift create cluster powervs \
+      --base-domain scnl-ibm.com \
+      --resource-group hypershift-resource-group \
+      --infra-id exmaple \
```

Comment:
If this section omits the `create infra` step above this is not needed, but should be included in the `Create cluster with infra created separately` example - also you have a typo in both places s/exmaple/example

Reply:
Fixed the commands in respective place, please take a look.
Fixed the typo.